src/cloud/env/environment-vars_magento.md

```diff
@@ -32,3 +32,6 @@ If you need to change the Admin URL after installation, use the `magento-cloud v
 ```bash
 magento-cloud variable:set ADMIN_URL magento_A8v10 -e master
```

Comment:
If that is the rule, then the underscore here is not valid?

Reply:
Hi, @hguthrie !

We have a validation rule for the custom admin path on the **admin** side only:

<img width="887" alt="magento_adm" src="https://user-images.githubusercontent.com/13585327/113876386-5f1a3480-97c0-11eb-8af2-c010e744e835.png">


However, a custom magento path may be set:
- in `app/etc/env.php` 
- in `ADMIN_URL` environment variable on Cloud (Cloud UI does not have any validation rules and we can type there everything we need):
<img width="814" alt="cloud_adm" src="https://user-images.githubusercontent.com/13585327/113876291-4447c000-97c0-11eb-975b-37455621a90e.png">

So, @EvheniiaKolkhiieva made good changes basic on codebase. In the same time our @rogyar is a very expirienced Magento specialist and he 100% knows that some additional chars (such as `_`) are also allowed in the custom admin path.

Thank you!

Reply:
Sweet! Thanks for the detailed response!
@atwixfirster @EvheniiaKolkhiieva  And thank you both for tackling this and clearing things up. Much appreciated.